relays/client-pangolin/src/lib.rs

```diff
@@ -0,0 +1,108 @@
+use std::time::Duration;
+
+// -- s2s bridger ---
+use codec::Encode;
+use sp_core::{Pair, storage::StorageKey};
+use sp_runtime::{
+	generic::SignedPayload,
+	traits::IdentifyAccount,
+};
+
+use relay_substrate_client::{Chain, ChainBase, ChainWithBalances, TransactionSignScheme};
+
+// --- darwinia ---
+use pangolin_runtime::*;
+use frame_support::traits::Len;
+
+/// Pangolin header id.
+pub type HeaderId = relay_utils::HeaderId<drml_primitives::Hash, drml_primitives::BlockNumber>;
+
+
+/// Rialto signing params.
+pub type SigningParams = sp_core::sr25519::Pair;
```

Comment:
This should be defined in common's pangolin primitives,

Reply:
https://github.com/darwinia-network/bridger/commit/fb7851a7a7947152642820ba0b5e458d0d993520